src/main/kotlin/org/openbase/bco/device/hass/manager/service/TemperatureStateServiceImpl.kt

```diff
@@ -0,0 +1,16 @@
+package org.openbase.bco.device.hass.manager.service
+
+import org.openbase.bco.device.hass.manager.dto.HassDeviceClass
+import org.openbase.bco.device.hass.manager.dto.HassStateAttributes
+import org.openbase.bco.device.hass.manager.dto.HassStateDto
+import org.openbase.type.domotic.state.TemperatureStateType.TemperatureState
+
+fun HassStateDto.isTemperatureSensor() : Boolean =
+    attributes[HassStateAttributes.DEVICE_CLASS.id] == HassDeviceClass.TEMPERATURE.id
```

Comment:
There should be a `.deviceClass()` extension `attributes[HassStateAttributes.deviceClass() == HassDeviceClass.TEMPERATURE.id`